lnbits/extensions/lnaddress/models.py

```diff
@@ -9,8 +9,8 @@
 class CreateDomain(BaseModel):
     wallet: str = Query(...)
     domain: str = Query(...)
-    cf_token: str = Query(...)
-    cf_zone_id: str = Query(...)
+    cf_token: str = Query("")
+    cf_zone_id: str = Query("")
```

Comment:
Got it. This might work: 

```suggestion
    cf_token: Optional[str] = Query("")
    cf_zone_id: Optional[str] = Query("")
```